Update skeletons.md (#552)
scala-js/scala-js-website: libraries/skeletons.md

These application skeletons get you started in no time!

### [Play application with Scala.js](https://github.com/vmunier/play-with-scalajs-example)

Example application showing how you can integrate [Play](https://www.playframework.com/) with Scala.js.

### [AWS Lambda with Scala.js](https://github.com/bgahagan/scalajs-lambda.g8)

A skeleton to build an AWS Lambda handler using Scala.js 1.0. Uses sbt-scalajs-bundler to manage npm dependencies and sbt-native-packager to package lambda for deployment.

### [npm scalajs-starter](https://github.com/konradst/scalajs-starter)

> Scala.js Node.js skeleton app / library handled with ```npm start```, ```build```, ```test```, ```clean``` commands.

Reviewer: This one seems outdated. It uses sbt-scalajs 1.0.0-M8.

Author: Indeed, I was keeping everything related to 1.0.0 as recent, I'll update the section

### [SPA tutorial with Scala.js and React](https://github.com/ochrons/scalajs-spa-tutorial)
Simple Single Page Application tutorial built upon scalajs-react, Play and Bootstrap.

### [Play application with Scala.js and Workbench](https://github.com/aholland/play-scalajs-workbench-example)
Example setup which, using the [Workbench plugin](https://github.com/lihaoyi/workbench), prevents any change made to client code from causing a compile error on the [Play](https://www.playframework.com/) server, which could otherwise slow down client development. 

### [Scala Google Spreadsheets](https://github.com/sherpal/Scala-Google-Spreadsheets)
Allows to create [Google custom functions](https://developers.google.com/apps-script/guides/sheets/functions) in [Google SpreadSheets](https://www.google.com/intl/en_UK/sheets/about/) with Scala.

## Archived

> These applications are not maintained anymore ubt could be useful to keep them as references.

Reviewer: ```suggestion
These applications are not maintained anymore but could be useful as references.
```

### [Node.js module with Scala.js](https://github.com/rockymadden/scala-node-example)

Proof of concept to determine if Scala.js could be leveraged to make a Node.js module.

### [Udash Generator](http://guide.udash.io/#/bootstrapping/generators)

> Project generator for the Udash framework - creates a customized, complete and runnable Udash based application.

Author: Udash-generator (which is the linked project) is archived: https://github.com/UdashFramework/udash-generator/

> Project generator for the Udash framework - creates a customized, complete and runnable Udash based application.

Author: Anyway, it is up to you what I do with this one

> 

Reviewer: Blank lines ;)
